edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Test/KubernetesDeploymentMapperTest.cs

```diff
@@ -139,8 +140,7 @@ public void EmptyDirMappingForVolume()
             var config = new KubernetesConfig("image", CreatePodParameters.Create(labels: labels, hostConfig: hostConfig), Option.None<AuthConfig>());
             var docker = new DockerModule("module1", "v1", ModuleStatus.Running, RestartPolicy.Always, Config1, ImagePullPolicy.OnCreate, DefaultConfigurationInfo, EnvVarsDict);
             var module = new KubernetesModule(docker, config);
-            var mapper = CreateMapper(storageClassName: null);
-
+            var mapper = new KubernetesDeploymentMapper("namespace", "edgehub", "proxy", Option.None<string>(), "configPath", "configVolumeName", "configMapName", "trustBundlePAth", "trustBundleVolumeName", "trustBundleConfigMapName", string.Empty, null, "apiVersion", new Uri("http://workload"), new Uri("http://management"));
```

Comment:
I think it's better to user CreateMapper method. If its signature lacks of some parameters lets add it